TransIT.DAL/Repositories/ImplementedRepositories/IssueLogRepository.cs

```diff
@@ -60,6 +60,7 @@ public IssueLogRepository(TransITDBContext context)
             .Include(d => d.OldState)
             .Include(e => e.Supplier)
             .Include(x => x.Document)
+            .Include(x => x.IssueLogParts)
```

Comment:
Why you are not including `Part` after including `IssueLogParts` ? How engineer would add parts to issuelog if he would not be able to see what he had already added. Look at other repositories changed and include entities.